qgis_hub_plugin/gui/resource_browser.py

```diff
@@ -53,6 +58,8 @@ def __init__(self, parent=None, iface=None):
         # Load resource for the first time
         self.populate_resources()
 
+        self.lineEditSearch.textChanged.connect(self.on_filter_text_changed)
+
         self.pushButtonDownload.clicked.connect(self.download_resource)
 
         self.checkBoxGeopackage.stateChanged.connect(
```

Comment:
I used lambda in case I am passing a parameter while calling the function.
In the new commits, I have removed the lambda where parameters are not passed to the function.